web/src/main/java/org/cbioportal/web/config/WebServletContextListener.java

```diff
@@ -0,0 +1,113 @@
+        this.oncokbPublicApiUrl = properties.getProperty("oncokb.public_api.url");
+
+        if (this.showOncokb) {
+            checkOncokbInfo();
+        }
+    }
+
+    @Override
+    public void afterPropertiesSet() throws Exception {
+    }
+
+    private void checkOncokbInfo() {
+        System.out.println("------------------------");
+        if(StringUtils.isEmpty(this.oncokbToken)) {
+            // oncokb.org is deprecated, www.oncokb.org should be used
+            System.out.println("OncoKB requires token to access the latest data. Please review oncokb terms(https://www.oncokb.org/terms) and register an account accordingly. For now, we will redirect the requests to a legacy instance which will not accept any new data changes. Thank you.");
+            LOG.warn("OncoKB requires token to access the latest data. Please review oncokb terms(https://www.oncokb.org/terms) and register an account accordingly. For now, we will redirect the requests to a legacy instance which will not accept any new data changes. Thank you.");
+        }
+    }
+
+    private InputStream getResourceStream(String propertiesFileName)
+    {
+        String resourceFilename = null;
+        InputStream resourceFIS = null;
+
+        try {
+            String home = System.getenv("PORTAL_HOME");
+            if (home != null) {
+                resourceFilename =
+                    home + File.separator + propertiesFileName;
+                if (LOG.isInfoEnabled()) {
+                    LOG.info("Attempting to read properties file: " + resourceFilename);
+                }
+                resourceFIS = new FileInputStream(resourceFilename);
+                if (LOG.isInfoEnabled()) {
+                    LOG.info("Successfully read properties file");
+                }
+            }
+        }
+        catch (FileNotFoundException e) {
+            if (LOG.isInfoEnabled()) {
+                LOG.info("Failed to read properties file: " + resourceFilename);
+            }
+        }
+
+        if (resourceFIS == null) {
+            if (LOG.isInfoEnabled()) {
+                LOG.info("Attempting to read properties file from classpath");
+            }
+            resourceFIS = this.getClass().getClassLoader().
+                getResourceAsStream(propertiesFileName);
+            if (LOG.isInfoEnabled()) {
+                LOG.info("Successfully read properties file");
+            }
+        }
+
+        return resourceFIS;
+    }
+    private Properties loadProperties(InputStream resourceInputStream)
+    {
+        Properties properties = new Properties();
+
+        try {
+            properties.load(resourceInputStream);
+            resourceInputStream.close();
+        }
+        catch (IOException e) {
+            if (LOG.isErrorEnabled()) {
+                LOG.error("Error loading properties file: " + e.getMessage());
+            }
+        }
+
+        return properties;
+    }
```

Comment:
I copied these code from `GlobalProperties`. Since we don't include core as the dependency of the web. Not sure how to what the convention is to share the code.